src/components/joinus/JoinUsPage.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import MarkDownRenderer from '../commons/renderer/MarkDownRenderer';
+import joinUsFile from './JoinUs.md';
+
+const JoinUsPage = () => {
+    const [markDownText, setMarkDownText] = useState('');
+
+    useEffect(() => {
+        async function fetchFileAndSetState() {
+            const response = await fetch(joinUsFile);
+            const text = await response.text();
```

Comment:
this await is really necessary? because, in my understanding response will have a value when arrive at this line
```suggestion
            const text = response.text();
```

Reply:
Fetch returns a `Response` object, and its body is a `Readable Stream`. In the same way we need to use `json` async method when we want this response's body as a JSON, in this case it's necessary to get the body as a text, so I decided to use `text` method.